### ISSUE TITLE
'NOT LIKE' query

### ISSUE THREAD

**Issue by kai23** [balderdashy/waterline#666]
Hello,

I want to perform a « not like » query
I tried something like this , in the find : 

	status: {
		'!': ['D_%']
	},

But it's always empty. Do you have any idea on how to do it ?

**Comment by dmarcelino** [comment]
Hi @kai23, I don't think this is supported currently. If you are using sails-mysql or sails-postgresql you should be able to this with `.query()`, which performs a raw SQL query.

**Comment by dmarcelino** [comment]
"Not contains" doesn't seem supported either: #22 and [“not contains” query in SailsJS Waterline](http://stackoverflow.com/questions/29362437/not-contains-query-in-sailsjs-waterline). I'm labelling this as a feature request.

**Comment by tjwebb** [comment]
Is your column nullable?

**Comment by kai23** [comment]
@tjwebb it's an enum, so it isn't.

**Comment by pmoelgaard** [comment]
I'm not entirely certain how general this solution is, however it works for me using Sails 0.11.0 against a Mongo v3.0.0 using sails-mongo 0.10.8:

```javascript
[model].destroy( {
            uid: { $nin: [array_of_uid] }
          })
```

**Comment by sailsbot** [comment]
Thanks for posting, @kai23.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!

**Comment by Joanne16** [comment]
Any update on this feature request ? It is very important for our project to be able to set 'not like' or 'not contains' options in the where clause (using .query() is not an option for the moment as we are using both my-sql and ms-sql adapters). The main reason we are using waterline is the ORM and query language features, where we don't need to duplicate query string for each database adapters.

Thanks
